Sources/General/BinaryMap.h

```diff
@@ -45,7 +47,14 @@ void BinaryMap_Clear(const BinaryMap *me);
 void BinaryMap_Invert(const BinaryMap *me);
 BinaryMap BinaryMap_GetInverted(const BinaryMap *me);
 bool BinaryMap_IsEmpty(const BinaryMap *me);
-void BinaryMap_Or(BinaryMap *me, const BinaryMap *them);
+void BinaryMap_Or(const BinaryMap *me, const BinaryMap *them);
 void BinaryMap_And(BinaryMap *me, const BinaryMap *them);
+void BinaryMap_AndArea(BinaryMap *me, const BinaryMap *source, const RectangleC *area, const Point *at);
```

Comment:
Should `BinaryMap_And` and `BinaryMap_AndArea`'s first BinaryMap parameter both be marked `const`, too?